src/Translator/src/Intl/ErrorKind.php

```diff
@@ -15,6 +15,8 @@
  * Adapted from https://github.com/formatjs/formatjs/blob/590f1f81b26934c6dc7a55fff938df5436c6f158/packages/icu-messageformat-parser/error.ts#L9-L77.
  *
  * @experimental
+ *
+ * @internal
```

Comment:
```suggestion
 * @internal
```

Let's remove `@experimental` everywhere you add "`@internal`" ... as internal classes won't be coverd by BC promise anyway :)